### DISCUSSION TITLE
download.imagemagick.org unavailable?

### DISCUSSION THREAD

**Discussion by jhogervorst** [ImageMagick/ImageMagick#4402] [General] [answered]
It seems that there are problems with `download.imagemagick.org`.

When I open such URLs ([example](https://download.imagemagick.org/ImageMagick/download/binaries/magick)) in the browser, I get an error message from my browser.

With some installation command, I get a similar error:
```
curl: (7) Failed to connect to download.imagemagick.org port 443: No route to host
```

**Comment by snibgo** [comment]
Your example is:
```
https://download.imagemagick.org/ImageMagick/download/binaries/magick
```
I don't know what link you want. If you remove the final "/magick", it works.

**Reply by jhogervorst** [reply]
When I posted the issue, the link above showed a browser error (about not being able to connect to the server). Right now it works again.

**Comment by jhogervorst** [comment, marked as answer]
It seems to work again now!

**Comment by AnuMano** [comment]
It seems imagemagic.org download is unavailable. 
Is anyone facing this issue?

**Comment by snibgo** [comment]
I currently have problems, eg downloading https://imagemagick.org/Usage/images/overlay_figure.gif . This gives a timeout message.

**Comment by snibgo** [comment]
It works fine today.
